.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,49 @@
+# More info:
+# https://docs.github.com/en/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning
+
+name: "CodeQL"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
```

Comment:
Would it be possible (with no disadvantages) to run for pull requests to all branches? It would be nice to not have to remember to add future release branches to this list.

Reply:
We can certainly make it run on all branches.